config/env.gitpod

```diff
@@ -0,0 +1,247 @@
+################################################################################
+# Environment Variables for Development in GitPod
+#
+# Copy this file to `.env` for use by docker-compose.
+# Copy this file to `.env` for use by docker-compose.
+################################################################################
+
+COMPOSE_PROJECT_NAME=telescope_api
+
+# Compose files to use together during development. NOTE: we specify separator below
+# so it will work on Windows and Unix, see
+# https://docs.docker.com/compose/reference/envvars/#compose_file
+COMPOSE_PATH_SEPARATOR=;
+COMPOSE_FILE=docker/docker-compose.yml;docker/gitpod.yml
```

Comment:
Here I think you it would be possible to use the `development.yml` file, so that your `gitpod.yml` can use it as a base.